apps/web/src/scenes/Onboarding/OnboardingAddProfilePicture.tsx

```diff
@@ -0,0 +1,51 @@
+import { useRouter } from 'next/router';
+import { useCallback } from 'react';
+import { flows } from 'shared/analytics/constants';
+
+import { VStack } from '~/components/core/Spacer/Stack';
+import { NftSelector } from '~/components/NftSelector/NftSelector';
+import useUpdateProfileImage from '~/components/NftSelector/useUpdateProfileImage';
+import FullPageCenteredStep from '~/components/Onboarding/FullPageCenteredStep';
+import { OnboardingFooter } from '~/components/Onboarding/OnboardingFooter';
+
+const onboardingStepName = 'add-profile-picture';
+export function OnboardingAddProfilePicture() {
```

Comment:
nit - maybe for consistency can rename to `OnboardingAddProfilePicturePage` since the other steps end with `Page`

Reply:
Updated the component naming!